Dockerfile.handle_dataset_submissions

```diff
@@ -0,0 +1,31 @@
+FROM public.ecr.aws/lambda/python:3.9
+
+COPY backend/__init__.py ./backend/__init.py
+COPY backend/corpora/__init__.py ./backend/corpora/__init__.py
+COPY backend/corpora/common ./backend/corpora/common
+COPY backend/corpora/dataset_submissions .
+
+# TODO: remove after the authz refactoring (_owner_or_allowed)
```

Comment:
These `COPY`s will be eliminated after Trent's auth code refactoring is merged in.